core/src/main/java/org/apache/calcite/avatica/AvaticaSite.java

```diff
@@ -291,7 +291,7 @@ public void setObject(Object x, int targetSqlType) {
 
   /** Similar logic to {@link #setObject}. */
   public static Object get(Cursor.Accessor accessor, int targetSqlType,
-      Calendar localCalendar) throws SQLException {
+                           boolean signed, Calendar localCalendar) throws SQLException {
```

Comment:
Nit: the indent, so are the following changes below in MetaImpl.

Reply:
Thank you, I have modified the indent.